internal/runtime/handler.go

```diff
@@ -205,13 +219,31 @@ func (h *Handler) getRuntimes(w http.ResponseWriter, req *http.Request) {
 			httputil.WriteErrorResponse(w, http.StatusInternalServerError, err)
 			return
 		}
-		err = h.setRuntimeOptionalAttributes(&dto, kymaConfig, clusterConfig, gardenerConfig)
+
+		instanceDrivenByKimOnly := h.kimConfig.IsDrivenByKimOnly(dto.ServicePlanName)
+
+		err = h.setRuntimeOptionalAttributes(&dto, kymaConfig, clusterConfig, gardenerConfig, instanceDrivenByKimOnly)
 		if err != nil {
 			h.logger.Warn(fmt.Sprintf("unable to set optional attributes: %s", err.Error()))
 			httputil.WriteErrorResponse(w, http.StatusInternalServerError, err)
 			return
 		}
 
+		if runtimeResourceConfig && dto.RuntimeID != "" && instanceDrivenByKimOnly {
```

Comment:
I think the `instanceDrivenByKimOnly` in not necessary here, I'd like to have a possibility to use kcp cli to get the runtime even if it is driven by Provisioner (in the phase when Provisioner is called and runtime resource is created).

Reply:
In the next commit/push.